lib/matplotlib/axes/_axes.py

```diff
@@ -3503,18 +3503,23 @@ def boxplot(self, x, notch=None, sym=None, vert=None, whis=None,
             everything is drawn horizontally.
 
         whis : float, sequence, or string (default = 1.5)
-            As a float, determines the reach of the whiskers beyond the
-            first and third quartiles. In other words, where IQR is the
-            interquartile range (`Q3-Q1`), the upper whisker will extend to
-            last datum less than `Q3 + whis*IQR`). Similarly, the lower whisker
-            will extend to the first datum greater than `Q1 - whis*IQR`.
-            Beyond the whiskers, data
-            are considered outliers and are plotted as individual
-            points. Alternatively, set
-            this to an ascending sequence of percentile (e.g., [5, 95])
-            to set the whiskers at specific percentiles of the data.
-            Finally, ``whis`` can be the string ``'range'`` to force the
-            whiskers to the min and max of the data.
+            The position of the whiskers.
+
+            If a float, the lower whisker is at position ``Q1 - whis*(Q3-Q1)``,
```

Comment:
To be precise,  the whisker extends at most that far, but ends at the last data point within this range

> the lowest datum still within 1.5 IQR of the lower quartile, and the highest datum still within 1.5 IQR of the upper quartile (often called the Tukey boxplot)

https://en.wikipedia.org/wiki/Box_plot#Types (second bullet point).